static/app/components/avatarChooser.tsx

```diff
@@ -281,8 +280,9 @@ const AvatarForm = styled('div')`
 
 const AvatarSubmit = styled('fieldset')`
   display: flex;
-  justify-content: flex-end;
-  margin-top: 1.25em;
+  align-items: center;
+  margin-top: 2.25em;
+  padding-top: 1.5em;
```

Comment:
I know this stuff already had em spacings, feel free to change over to `space` if you want